examples/ros-service-call-module/src/JsonSchemaForm/BooleanInput.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Typography, Switch } from "@formant/ui-sdk";
+import React, { FC } from "react";
+interface INumberInputProps {
+  jsonSchemaObject: any;
+  currentStateObject: any;
+  property: string;
+}
+
+export const BooleanInput: FC<INumberInputProps> = ({
+  jsonSchemaObject,
+  currentStateObject,
+  property,
+}) => {
+  return (
+    <Box display="flex" height={56} alignItems="center" marginBottom={2}>
+      <Typography>
+        {jsonSchemaObject.properties[property].title[0].toUpperCase() +
+          jsonSchemaObject.properties[property].title.slice(1) +
```

Comment:
given the number of times this behvaior is used, it wouldn't _hurt readability_ to extract out a `capitalize()` method (or just use [`lodash.captialize()`](https://www.npmjs.com/package/lodash.capitalize))